Source/WebKit/Platform/IPC/StreamConnectionEncoder.h

```diff
@@ -60,15 +57,17 @@ class StreamConnectionEncoder final {
     bool encodeSpan(std::span<T, Extent> span)
     {
         auto bytes = asBytes(span);
-        size_t bufferPointer = static_cast<size_t>(reinterpret_cast<intptr_t>(m_buffer + m_encodedSize));
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+        size_t bufferPointer = static_cast<size_t>(reinterpret_cast<intptr_t>(m_buffer.data() + m_encodedSize));
```

Comment:
I think you can do m_buffer.subspan(m_encodedSize).data() to eliminate the skip markers (and gain a bound check).

Reply:
Yes, this was my initial attempt. However, `m_buffer.subspan(m_encodedSize)` hits the assertion in subspan() so I'm leaving it alone for now.